packages/yoga/src/List/web/LinkItem.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledLinkItem = styled.a`
+  ${({
+    theme: {
+      yoga: {
+        components: { list },
+      },
+    },
+  }) => `
+    padding:
+      ${list.listItem.padding.top}px
+      ${list.listItem.padding.right}px
+      ${list.listItem.padding.bottom}px
+      ${list.listItem.padding.left}px;
+  `}
+
+  display: inline-block;
+
+  width: 100%;
+  height: 100%;
+
+  text-decoration: none;
```

Comment:
We use this callback in the end of all of styled component's rules.

```suggestion
  display: inline-block;

  width: 100%;
  height: 100%;

  text-decoration: none;
  
  ${({
    theme: {
      yoga: {
        components: { list },
      },
    },
  }) => `
    padding:
      ${list.listItem.padding.top}px
      ${list.listItem.padding.right}px
      ${list.listItem.padding.bottom}px
      ${list.listItem.padding.left}px;
  `}
```